projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -732,6 +738,95 @@ export function insertCommentAboveIdentifier(
   return changes;
 }
 
+export function getImportDeclarations(
```

Comment:
Do we need this exported?
Also, could we somehow merge logic from `getImportDeclarationNode` and this method?

Reply:
Fixed. Used this method in `getImportDeclarationNode`.